src/pages/guides/runtime_webhooks.md

```diff
@@ -28,7 +28,7 @@ The Runtime cli will let you create a runtime action and hook it up with an inte
 
 ### Built In Signature Verification
 
-With integration between I/O Events and Adobe I/O Runtime, you don't need to worry about security as your runtime actions configured as webhooks are secured with an out-of-the-box signature verification implementation. So, basically whatever runtime action you use to create an event registration, the handler webhook created due to that will place a signature validator action along with your action as in a [sequence](/runtime/docs/guides/reference/sequences_compositions/). I/O Events signs the event payload using digital signatures genereated using its private keys and passes the signatures as well as its public keys as request headers while invoking your webhook. Your business logic runtime action will only be invoked once the validator action successfully verifies any one of the digital signatures using the public keys. You can read more about the digital signatures based authentication mechanism [here](./index.md).
+With integration between I/O Events and Adobe I/O Runtime, you don't need to worry about security as your runtime actions configured as webhooks are secured with an out-of-the-box signature verification implementation. So, basically whatever runtime action you use to create an event registration, the handler webhook created due to that will place a signature validator action along with your action as in a [sequence](/runtime/docs/guides/reference/sequences_compositions/). I/O Events signs the event payload using digital signatures generated using its private keys and passes the signatures as well as its public keys as request headers while invoking your webhook. Your business logic runtime action will only be invoked once the validator action successfully verifies any one of the digital signatures using the public keys. You can read more about the digital signatures based authentication mechanism [here](./index.md#security-considerations).
```

Comment:
For some reason, the relative link does not work with an anchor in gatsby. Try with `npm run dev`. Absolute link works though.

Reply:
okay. let me change that